lib/chef/http/authenticator.rb

```diff
@@ -16,16 +16,18 @@
 # limitations under the License.
 #
 
+require "chef/mixin/powershell_exec"
 require_relative "auth_credentials"
 require_relative "../exceptions"
 autoload :OpenSSL, "openssl"
 
 class Chef
   class HTTP
     class Authenticator
-
       DEFAULT_SERVER_API_VERSION = "2".freeze
 
+      include Chef::Mixin::PowershellExec
```

Comment:
you're using powershell_exec in a class method below so I think this needs to be `extend`